Sources/SwiftIfConfig/SyntaxProtocol+IfConfig.swift

```diff
@@ -82,20 +54,10 @@ extension SyntaxProtocol {
   /// If you are querying whether many syntax nodes in a particular file are
   /// active, consider calling `configuredRegions(in:)` once and using
   /// this function. For occasional queries, use `isActive(in:)`.
+  @available(*, deprecated, message: "Please use ConfiguredRegions.isActive(_:)")
```

Comment:
Is anyone using this yet (the compiler?). I’d be fine with removing it straight away since we don’t have a release with that API yet.

Reply:
Nobody is using it; we can zap it now.